src/gauge.js

```diff
@@ -251,39 +273,104 @@
           d: pathString(radius, startAngle, startAngle) // value of 0
         });
 
+        if (showValueCursor) {
+          gaugeValueCursorLinePath = svg("line", {
+            fill: "none",
+            stroke: "black",
+            className: "cursor-line",
+            opacity: "0",
+            "stroke-dasharray": "2",
+            "stroke-dashoffset": "0.5",
+            "stroke-width": 0.5,
+          });
+
+          gaugeValueCursorTrianglePath = svg("path", {
+            fill: "black",
+            stroke: "black",
+            opacity: "0",
+            "stroke-width": 0.5
+          });
+
+          var lineOpacityTransition = "opacity 0.5s";
+          var lineOpacityTransitionDelay = "0.8s";
+
+          gaugeValueCursorLinePath.style["-webkit-transition"] = lineOpacityTransition;
+          gaugeValueCursorLinePath.style["-moz-transition"] = lineOpacityTransition;
+          gaugeValueCursorLinePath.style["transition"] = lineOpacityTransition;
+          gaugeValueCursorLinePath.style["transition-delay"] = lineOpacityTransitionDelay;
+
+          var triangleOpacityTransition = "opacity 0.5s";
+          var triangleOpacityTransitionDelay = "1.6s";
+
+          gaugeValueCursorTrianglePath.style["-webkit-transition"] = triangleOpacityTransition;
+          gaugeValueCursorTrianglePath.style["-moz-transition"] = triangleOpacityTransition;
+          gaugeValueCursorTrianglePath.style["transition"] = triangleOpacityTransition;
+          gaugeValueCursorTrianglePath.style["transition-delay"] = triangleOpacityTransitionDelay;
+        }
+
         var angle = getAngle(100, 360 - Math.abs(startAngle - endAngle));
         var flag = angle <= 180 ? 0 : 1;
-        var gaugeElement = svg("svg", {"viewBox": viewBox || "0 0 100 100", "class": gaugeClass}, [
-          svg("path", {
-            "class": dialClass,
-            fill: "none",
-            stroke: "#eee",
-            "stroke-width": 2,
-            d: pathString(radius, startAngle, endAngle, flag)
-          }),
+
+        var gaugeDialPath = svg("path", {
+          "class": dialClass,
+          fill: "none",
+          stroke: "#eee",
+          "stroke-width": 2,
+          d: pathString(radius, startAngle + dialAnglePadding, endAngle - dialAnglePadding, flag)
+        });
+
+        var gaugeElement = svg("svg", { "viewBox": viewBox || "0 0 100 100", "class": gaugeClass }, [
+          !valueToBackground && gaugeDialPath,
+          gaugeValuePath,
           svg("g", { "class": "text-container" }, [gaugeValueElem]),
-          gaugeValuePath
-        ]);
+          valueToBackground && gaugeDialPath,
+          showValueCursor && gaugeValueCursorLinePath,
+          showValueCursor && gaugeValueCursorTrianglePath
+        ].filter(x => x));
```

Comment:
They don't use arrow function in this repo, maybe do this:
```suggestion
        ].filter(Boolean));
```